sympy/combinatorics/pc_groups.py

```diff
@@ -75,63 +76,55 @@ def __init__(self, pc_relators, relative_order, group):
         self.relative_order = relative_order
         self.group = group
 
-    def minimal_uncollected_subwords(self, word):
+    def minimal_uncollected_subword(self, word):
         """
         Returns the minimal uncollected subwords.
 
         Examples
         ========
         >>> from sympy.combinatorics.pc_groups import Collector
         >>> from sympy.combinatorics.free_groups import free_group
+        >>> from sympy.core import S
 
         Example 8.7 Pg. 281 from [1]
         >>> F, x1, x2 = free_group("x1, x2")
         >>> pc_relators = {x1**2 : (), x1*x2*x1**-1 : x2**-1, x1**-1*x2*x1 : x2**-1}
-        >>> relative_order = {x1: 2, x2: 3}
+        >>> relative_order = {x1: 2, x2: S.Infinity}
```

Comment:
I think that we should avoid importing `S.Infinity`. It would suffice to just omit `x2` from the relative_order dict.